Src/AutoFixture/Kernel/NoSpecimen.cs

```diff
@@ -93,10 +65,6 @@ public bool Equals(NoSpecimen other)
             {
                 return false;
             }
```

Comment:
this method should probably be
```c#
return other is { };
```

Reply:
Will make it `return other is not null;` as I think that is more readable idiom.